pkg/s3select/select.go

```diff
@@ -405,3 +452,25 @@ func aggregationFunctions(counter int, filtrCount int, myAggVals []float64, colu
 	}
 	return nil
 }
+
+// convertToSlice takes the map[string]interface{} and convert it to []string
+func convertToSlice(columnsMap map[string]int, record map[string]interface{}, marshalledRecord string) []string {
```

Comment:
```
convertToSlice()
```
why is this function needed?

Reply:
The input type is whether JSON/CSV is read as map[string]interface. Now when the output is csv , the marshalled record is converted to  slice in the particular order , which is read from `columnsMap map[string]int` which keeps the sequencing of column intact.